pom.xml

```diff
@@ -633,9 +627,18 @@
         <groupId>joda-time</groupId>
         <artifactId>joda-time</artifactId>
         <version>${joda-time.version}</version>
-    </dependency>
+      </dependency>
     </dependencies>
   </dependencyManagement>
+
+  <!-- Added SLF4J dependency for SW360ServiceContextListener -->
+  <dependencies>
+    <dependency>
+      <groupId>org.slf4j</groupId>
+      <artifactId>slf4j-api</artifactId>
+    </dependency>
+  </dependencies>
+
```

Comment:
Already listed in the file https://github.com/eclipse-sw360/sw360/blob/e464254be58671678b29fd896bf8bddf8f81810a/pom.xml#L231-L235

Please revert irrelevant changes.